code/modules/reagents/bonsai.dm

```diff
@@ -54,4 +55,9 @@
 					"wheat",
 					"potato",
 					"rice")]
-				S.harvest(get_turf(src),0,0,1)
\ No newline at end of file
+				S.harvest(get_turf(src),0,0,1)
+
+/obj/item/reagent_containers/bonsai/New()
```

Comment:
25th line, you have New() used twice (this breaks a lot of things). You need to remove this New() and put icon_state into already existing one.

Reply:
i dont understand what you mean, this breaks what?

Reply:
did what you told me in discord, should be ok i hope